src/App.ts

```diff
@@ -1,55 +1,83 @@
-import { Application } from 'pixi.js';
-
-interface Settings {
-    background: {
-        color: string,
-    },
-    dimensions: {
-        width: number,
-        height: number,
-    },
-}
+import { AppConfig } from "./common/AppConfig";
+import { Application, ApplicationOptions } from "pixi.js";
+import { Config } from "./App.types";
+import { DEFAULT_MANIFEST } from "./App.consts";
+import { Events } from "./common/Events";
+import { HomePage } from "./ui/pages/HomePage";
```

Comment:
This Homepage component hasn't been used in this file and gives an error when tying to compile

Reply:
fixed